node/CHANGELOG.md

```diff
@@ -155,4 +154,4 @@ All notable changes to this project will be documented in this file.  The format
 [1.1.1]: https://github.com/casper-network/casper-node/compare/v1.0.1...v1.1.1
 [1.1.0]: https://github.com/casper-network/casper-node/compare/v1.0.1...v1.1.1
 [1.0.1]: https://github.com/casper-network/casper-node/compare/v1.0.0...v1.0.1
-[1.0.0]: https://github.com/casper-network/casper-node/releases/tag/v1.0.0
\ No newline at end of file
+[1.0.0]: https://github.com/casper-network/casper-node/releases/tag/v1.0.0
```

Comment:
Did anything actually change in this line??

Reply:
Just a newline which my editor automatically applies at the end of all files on saving.